pkg/configuration/configuration_test.go

```diff
@@ -60,6 +60,14 @@ func TestGetAllRegistrationServiceParameters(t *testing.T) {
 	})
 }
 
+func TestEnvironment(t *testing.T) {
+	restore := test.SetEnvVarAndRestore(t, "HOST_OPERATOR_ENVIRONMENT", "e2e-test")
+	defer restore()
+
+	config := getDefaultConfiguration(t)
+	assert.Equal(t, "e2e-test", config.GetEnvironment())
```

Comment:
Also check that if no `HOST_OPERATOR_ENVIRONMENT` set then `prod` is used.

Reply:
good idea, updated here: 9e8605c8ad04454dac98359c92913e9abdedc64d